pkg/handlers/app.go

```diff
@@ -141,40 +139,41 @@ func GetAppUpdates(w http.ResponseWriter, r *http.Request) {
 	}
 
 	license := store.GetStore().GetLicense()
+	updates := store.GetStore().GetUpdates()
 
-	if !util.IsAirgap() {
```

Comment:
why don't we need this anymore?

Reply:
you can't check for updates in airgap mode :)

Reply:
isn't `if !util.IsAirgap() { ... }` what prevented that before? someone can hit this endpoint in airgap mode and it will try to make outbound requests. lmk if i'm misreading the diff